mautic/mautic
# TPROD-322 - PageBundle - Convert PHP Templates to Twig Templates

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. 



- Does searching for a page appear in the global search?
- /s/pages work? Sort, buttons, etc.
- Does the results page work? filter, sort
- [results] export as csv, excel, html?
- Details page load correctly
- [details] has A/B tests
- [details] has translations
- [details] A/B test stats modal work correctly
- Preview page loads correctly
- Add A/B test to existing landing page
- Create new landing page
- Edit existing landing page
- Does the builder function correctly?
- Within Builder, do all slot types work correctly?
- Within builder, do all section types work correctly?
